pkg/motor/query.go

```diff
@@ -80,3 +80,17 @@ func (mtr *motorNodeImpl) QuerySchema(req mt.QueryWhatIsRequest) (*mt.QueryWhatI
 		WhatIs: mtr.Resources.whatIsStore[req.Did],
 	}, nil
 }
+
+func (mtr *motorNodeImpl) QuerySchemaGroup(req mt.QueryWhatIsByCreatorRequest) (*mt.QueryWhatIsByCreatorResponse, error) {
+	whatIss, err := mtr.GetClient().QueryWhatIsByCreator(req.Creator)
+
+	if err != nil {
+		return nil, err
+	}
+	resp := mt.QueryWhatIsByCreatorResponse{
+		Code:   http.StatusAccepted,
```

Comment:
nit: StatusOK is more appropriate here

Reply:
I have some similar changes in my current branch so I'll go ahead and merge this and fix it there